packages/integration/config.d.ts

```diff
@@ -115,11 +115,16 @@ export interface Config {
        * @visibility secret
        */
       username: string;
+      /**
+       * Token used to authenticate requests.
+       * @visibility secret
+       */
+      token?: string;
       /**
        * Bitbucket Cloud app password used to authenticate requests.
        * @visibility secret
```

Comment:
```suggestion
       * @visibility secret
       * @deprecated Use `credentials` instead.
```

Can we do this as well? Make it easier for when we want to remove this.

Reply:
credentials or token?

Reply:
Token sorry, copy and paste error on my part :(